packages/usa-banner/src/test/template.html

```diff
@@ -9,38 +9,43 @@
           <p class="usa-banner__header-text">An official website of the United States government</p>
           <p class="usa-banner__header-action" aria-hidden="true">Here’s how you know</p>
         </div>
-        <button class="usa-accordion__button usa-banner__button"
-          aria-expanded="false"
-          aria-controls="gov-banner"
-        >
-          <span class="usa-banner__button-text">Here's how you know</span>
+        <button class="usa-accordion__button usa-banner__button" aria-expanded="false"
+          aria-controls="gov-banner-default">
+          <span class="usa-banner__button-text">Here’s how you know</span>
         </button>
       </div>
     </header>
-    <div class="usa-banner__content usa-accordion__content" id="gov-banner">
+    <div class="usa-banner__content usa-accordion__content" id="gov-banner-default" hidden="">
       <div class="grid-row grid-gap-lg">
```

Comment:
I see grid references here that were removed in usa-banner.twig. Should they be removed in this file as well?

Reply:
Totally, removed in b50e5a6eb3d24893d13310e7290b42d8da683e18. Good catch!